extmod/extmod.mk

```diff
@@ -514,3 +515,90 @@ include $(TOP)/extmod/btstack/btstack.mk
 endif
 
 endif
+
+################################################################################
+# openamp
+
+ifeq ($(MICROPY_PY_OPENAMP),1)
+OPENAMP_DIR = lib/open-amp
+LIBMETAL_DIR = lib/libmetal
+GIT_SUBMODULES += $(LIBMETAL_DIR) $(OPENAMP_DIR)
+
+INC += -I$(TOP)/$(OPENAMP_DIR)
+CFLAGS += -DMICROPY_PY_OPENAMP=1
+
+# libmetal is intended to run through a pre-processor (as part of its CMake-based build system).
```

Comment:
Can we move this block (all the sed lines and SRC_LIBMETAL_C etc..) to `extmod/libmetal/metal.mk` ?

Reply:
Yes that would be good.

Reply:
Yes, that sounds fine.  All things related to libmetal configuration can go there.

Reply:
Done.